scripts/VTS/IRF.evndisp_MC.sh

```diff
@@ -80,11 +80,12 @@ ZA=$4
 WOBBLE=$5
 NOISE=$6
 SIMTYPE=$7
-[[ "$8" ]] && ACUTS=$8 || ACUTS=EVNDISP.reconstruction.runparameter
+[[ "$8" ]] && ACUTS=$8 || ACUTS=EVNDISP.reconstruction.runparameter.pSCT
```

Comment:
That change is very special for the pSCT analysis and should not be merged with master